src/app/visit/visit.component.ts

```diff
@@ -38,25 +38,23 @@ export class VisitComponent implements OnInit {
         'type',
         'image',
     ];
-    dataSource:
-        | Store[]
-        | MatTableDataSource<Store, MatTableDataSourcePaginator>;
+    dataSource: MatTableDataSource<Store, MatTableDataSourcePaginator>;
     visitPoints: number[] = [];
+    store: Store[];
     selected = false;
+    checked = false;
```

Comment:
what is `checked` for what is used?

Reply:
it used for hiding and showing check and finish button but i change this names